sympy/stats/drv.py

```diff
@@ -69,20 +69,21 @@ def _pmf(self, x):
 class SampleDiscreteNumpy:
     """Returns the sample from numpy of the given distribution"""
 
-    def __new__(cls, dist, size):
-        return cls._sample_numpy(dist, size)
+    def __new__(cls, dist, size, seed=None):
+        return cls._sample_numpy(dist, size, seed)
 
     @classmethod
-    def _sample_numpy(cls, dist, size):
+    def _sample_numpy(cls, dist, size, seed):
         """Sample from NumPy."""
 
         import numpy
+        rand_state = numpy.random.RandomState(seed=seed)
```

Comment:
This should be
```suggestion
        rand_state = numpy.random.default_rng(seed)
```
As numpy docs claim the default_rng (using PCG64) has better statistical properties than the mersenne twister of the RandomState.